### DISCUSSION TITLE
Oil Spill assessment using Sentinel 1 products

### DISCUSSION THREAD

**Discussion by bbristold** [semiautomaticgit/SemiAutomaticClassificationPlugin#229] [General]
Please how can we use this Qgis Plugin to detect the oil Spill in the SAR images?

**Comment by semiautomaticgit** [comment]
Hello @bbristold , I don't have specific documentation about this topic.
I suggest to read this tutorial that describes how to perform a classification, given that you already have SAR raster data https://semiautomaticclassificationmanual.readthedocs.io/en/latest/tutorial_1.html

**Comment by bbristold** [comment]
Thanks 👍
I need to detect the oil spill using this SCP.

Le dim. 17 juil. 2022 à 17:37, Luca Congedo ***@***.***> a
écrit :
